src/main/java/Baron.java

```diff
@@ -0,0 +1,92 @@
+import java.util.Scanner;
+
+public class Baron {
+    public static void displayCommandsList(Task[] commandsList) {
+        int taskNumber = 1;
+        System.out.println("\tHere are the tasks in your list:");
+        for (Task command: commandsList) {
+            if (command == null) {
+                break;
+            }
+            System.out.println("\t" + taskNumber + "." + command.getTaskDetails());
+            taskNumber += 1;
+        }
+        System.out.println();
+    }
+
+    public static void main(String[] args) {
+        greetUser();
+        getUserInput();
+        printExitMessage();
+    }
+
+    private static void greetUser() {
+        System.out.println("Hello! I'm Baron");
+        System.out.println("What can I do for you?\n");
+    }
+
+    private static void printExitMessage() {
+        System.out.println("Bye. Hope to see you again soon!");
+    }
+
+    private static void getUserInput() {
+        Task[] tasks = new Task[100];
+        int numberOfTasks = 0;
+        while (true) {
+            Scanner userInput = new Scanner(System.in);
+            String input = userInput.nextLine();
+            input = input.toLowerCase();
+            String[] inputArray = input.split(" ", 2);
+            if (input.equals("bye")) {
+                break;
```

Comment:
Is there a need to use break in if else conditionals? since your code already checks for a very specific bound